alexia/apps/scheduling/management/commands/remindermail.py

```diff
@@ -12,12 +12,13 @@ def handle(self, *args, **options):
         for mailtemplate in MailTemplate.objects.filter(name=MailTemplate.REMINDER, is_active=True):
```

Comment:
better, but is it not possible to move the active check to this for loop? `organization__is_active=True` or smth?